Packs/Jira/Integrations/JiraV2/JiraV2.py

```diff
@@ -729,51 +881,324 @@ def fetch_incidents(query, id_offset, fetch_by_created=None, **_):
+
+        else:
+            demisto.debug(f'Skipping updating remote incident fields [{remote_id}] '
+                          f'as it is not new nor changed')
+
+        if entries:
+            for entry in entries:
+                demisto.debug(f'Sending entry {entry.get("id")}, type: {entry.get("type")}')
+                if entry.get('type') == 3:
+                    demisto.debug('Add new file\n')
+                    path_res = demisto.getFilePath(entry.get('id'))
+                    file_name = path_res.get('name')
+                    upload_file(entry.get('id'), remote_id, file_name)
+                else:  # handle comments
+                    demisto.debug('Add new comment\n')
+                    add_comment(remote_id, str(entry.get('contents', '')))
+    except Exception as e:
+        demisto.error(f"Error in Jira outgoing mirror for incident {remote_args.remote_incident_id} \n"
+                      f"Error message: {str(e)}")
+    finally:
+        return remote_id
+
+
+def get_user_info_data():
+    """
+    This function returns details for a current user in order to get timezone.
+    :return: API response
+    """
+    HEADERS['Accept'] = "application/json"
+    return requests.request(method='GET', url=BASE_URL + 'rest/api/latest/myself', headers=HEADERS, verify=USE_SSL,
+                            auth=get_auth())
+
+
+def get_modified_remote_data_command(args):
+    """
+    available from Cortex XSOAR version 6.1.0. This command queries for incidents that were modified since the last
+    update. If the command is implemented in the integration, the get-remote-data command will only be performed on
+    incidents returned from this command, rather than on all existing incidents.
+    :param args: args['last_update']: Date string represents the last time we retrieved modified incidents for this
+     integration.
+    :return: GetModifiedRemoteDataResponse: this is the object that maintains a list of incident ids to run
+     'get-remote-data' on.
+    """
+    remote_args = GetModifiedRemoteDataArgs(args)
+    modified_issues_ids = []
+    HEADERS['Accept'] = "application/json"
+    try:
+        res = get_user_info_data()
+    except Exception as e:
+        demisto.error(f'Could not get Jira\'s timezone for get-modified-remote-data. failed because: {e}')
+    else:
+        if res.status_code == 200:
+            timezone_name = res.json().get('timeZone')
+            if not timezone_name:
+                demisto.error(f'Could not get Jira\'s time zone for get-modified-remote-data.Got unexpected reason:'
+                              f' {res.json()}')
+            last_update: datetime = parse(remote_args.last_update, settings={'TIMEZONE': timezone_name})\
+                .strftime('%Y-%m-%d %H:%M')
+            demisto.debug(f'Performing get-modified-remote-data command. Last update is: {last_update}')
+            _, _, context = issue_query_command(f'updated > "{last_update}"', max_results=50)
```

Comment:
```suggestion
            _, _, context = issue_query_command(f'updated > "{last_update}"', max_results=100)
```

also, I think we need to combine the query parameter with `updated > "{last_update}"`
like what we do in fetch_incidents